### PR TITLE
Switch search result icons to SVG and update icons

### PR DESCRIPTION
**Pull request by dvdmrtnz**
I have managed to recreate the search result rounded blue icon in SVG, including the drop shadow and the blue color gradient. This will allow us to remove all `search-result-xxxx` PNG icons in all resolutions from `data/styles/default/light/mdpi`, `hdpi`, `xhdpi`... and just keep one SVG file per icon in `data/styles/default/light/symbols` as with the usual POI icons. 

| PNG | SVG |
|-|-|
| search-result-pub | search-result-beer |

To do: 

- [x] Convert all search results icons to SVG

---

* Fixes https://github.com/organicmaps/organicmaps/issues/9218

### CODE REVIEW

**Comment by dvdmrtnz** [comment]
I've finished converting all search results icons to SVG.

This is ready for review.

File: map/search_mark.cpp [hunk 1]
```diff
@@ -151,37 +163,48 @@ class SearchMarkTypeChecker
       {{"shop", "beverages"},        SearchMarkType::ShopAlcohol},
       {{"shop", "wine"},             SearchMarkType::ShopAlcohol},
       {{"shop", "butcher"},          SearchMarkType::ShopButcher},
-      {{"shop", "boutique"},         SearchMarkType::ShopClothes},
```
Comment (vng) [inline map/search_mark.cpp:154]: Hm, I've just checked wiki and this type is deprecated and should be replaced with shop=clothes.
ToDo: Replace it in the classifier later?

File: map/search_mark.cpp [hunk 2]
```diff
@@ -208,9 +231,7 @@ SearchMarkPoint::SearchMarkPoint(m2::PointD const & ptOrg)
 
 m2::PointD SearchMarkPoint::GetPixelOffset() const
 {
-  if (!IsHotel())
-    return {0.0, 4.0};
-  return {0.0, 0.0};
+  return {0.0, 4.0};
```
Comment (vng) [inline map/search_mark.cpp:234]: Any ideas why it was made like this?

File: map/search_mark.cpp [hunk 3]
```diff
@@ -226,9 +247,7 @@ drape_ptr<df::UserPointMark::SymbolNameZoomInfo> SearchMarkPoint::GetSymbolNames
 
 drape_ptr<df::UserPointMark::SymbolOffsets> SearchMarkPoint::GetSymbolOffsets() const
 {
-  m2::PointF offset(0, 0);
-  if (!IsHotel())
-    offset.y = 1;
+  m2::PointF offset(0, 1);
```
Comment (vng) [inline map/search_mark.cpp:250]: Any ideas why it was made like this?

**Review by vng — APPROVED** [review]

File: map/search_mark.cpp (hunk 2 above)
Comment (dvdmrtnz) [reply map/search_mark.cpp:234]: Hotels icons were centered inside the PNG. The rest of the icons had a shadow so they were not centered in the PNG. That's why there was a different offset. 

After this PR all icons have a shadow and are offset the same in the SVG.

**Comment by soshial** [comment]
Will these new icons be used in Android as well?

**Comment by dvdmrtnz** [comment]
> Will these new icons be used in Android as well?

Yes, these are common between iOS/Android

**Pull request merged**